### ISSUE TITLE
undefined sym: __builtin_mul

### ISSUE THREAD

**Issue by aaronryank** [shinh/elvm#45]
Frequently when using the [online demo](http://shinh.skr.jp/elvm/8cc.js.html), attempting to assemble the ELVM IR just prints:

    undefined sym: __builtin_mul

It seems to occur anytime multiplication is needed. For example, the code below throws the error above:

    int main(void) { printf("%d", 6 * 3); }

**Comment by shinh** [comment]
I know this is a weird limitation, but you need #include <stdio.h>, which lets the compiler include https://github.com/shinh/elvm/blob/master/libc/_builtin.h .

Including this file by default would significantly slow down the compilation for minimal code. Maybe we should let compiler emit __builtin_xxx when xxx is used first time.

**Comment by aaronryank** [comment]
I agree, I think `_builtin.h` should be included by default. I can't think of many useful programs that won't use multiplication in some form.
